docker-compose.yml

```
@@ -1055,12 +1055,14 @@ services:
       LANG: "C.UTF-8"
       BUILD_DOCS_CPP: "ON"
       BUILD_DOCS_PYTHON: "ON"
+      PYTEST_ARGS: "--doctest-modules --doctest-cython"
     volumes: *conda-volumes
     command:
       ["/arrow/ci/scripts/cpp_build.sh /arrow /build &&
         /arrow/ci/scripts/python_build.sh /arrow /build &&
         pip install -e /arrow/dev/archery[numpydoc] &&
-        archery numpydoc --allow-rule PR01,PR10"]
+        archery numpydoc --allow-rule PR01,PR10
```

I think the `&&` will be required to be treated as two different commands. Otherwise both lines will be treated as a single command:
```suggestion
        archery numpydoc --allow-rule PR01,PR10 &&
```